.github/workflows/OperatorIntegrtionTest.yml

```diff
@@ -0,0 +1,163 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: .github/workflows/OperatorIntegrtionTest.yml
+env:
+  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  ECR_OPERATOR_STAGING_IMAGE: ${{ secrets.ECR_OPERATOR_STAGING_IMAGE }}
+
+on:
+  push:
+    branches:
+      - main*
+      - addOperatorIntgTest
```

Comment:
should just be main*. I assume `addOperatorIntgTest` was added for testing.

Reply:
yes.